src/main/resources/com/parasoft/findings/jenkins/coverage/ParasoftCoverageRecorder/config.properties

```diff
@@ -0,0 +1,3 @@
+title.pattern=Report File Pattern
+description.pattern=<a rel="noopener noreferrer" href="{0}">Fileset ''includes''</a> syntax \
+    specifying the Parasoft coverage files to read, such as ''**/coverage.xml''.
```

Comment:
Please check the possibility to display this default(**/coverage.xml) value as initial value of the input field.

Reply:
The default pattern value(`**/coverage.xml`) is set as the initial value and placeholder in the input. And if the input is empty, the pattern will be saved with the default pattern value(`**/coverage.xml`).